src/core/entities/post.entity.ts

```diff
@@ -0,0 +1,38 @@
+import "reflect-metadata";
+
+import { Field, Int, ObjectType } from "type-graphql";
+
+import {
+  //   BooleanProfileField,
+  //   NumberProfileField,
+  //   ProfilePrivacyField,
+  //   ProfilePrivacyFieldInput,
+  StringProfileField,
+} from "./profile.entity";
+
+@ObjectType()
+export class Post {
+  @Field(() => Int)
+  id!: number;
+
+  @Field()
+  userId?: number;
+
+  @Field(() => StringProfileField, { nullable: true })
+  content!: StringProfileField;
+
+  @Field()
+  image?: string;
+
+  @Field()
+  likes!: number;
+
+  @Field()
+  dislikes!: number;
+
+  @Field()
+  isDraft?: boolean;
+
+  @Field()
```

Comment:
Same with this one, should be required, default to false. Usually with booleans we can set a default since there are only 2 options, leaving it possibly null adds an extra option which will only confuse things later.